package.json

```diff
@@ -52,7 +52,7 @@
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.13",
-    "cypress": "^12.14.0",
+    "cypress": "^13.16.1",
```

Comment:
this is not related to the PR of course but i don't touch these tests often so wanted to take this opportunity to upgrade